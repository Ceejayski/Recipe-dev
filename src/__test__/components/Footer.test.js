import { BrowserRouter } from 'react-router-dom';
import { create } from 'react-test-renderer';
import Footer from '../../components/footer';

test('renders Correctly', () => {
  const Placeholder = () => (
    <BrowserRouter>
      <Footer />
    </BrowserRouter>
  );
  const tree = create(<Placeholder />).toJSON();
  expect(tree).toMatchSnapshot();
});
