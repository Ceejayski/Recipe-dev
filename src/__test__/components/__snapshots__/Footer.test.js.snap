// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Correctly 1`] = `
<div
  className="footer-basic d-flex justify-content-center text-center bg-white mt-5"
>
  <footer>
    <div
      className="social-links mb-2"
    >
      <a
        className="links-social"
        href="/"
      >
        <i
          aria-label="Social"
          className="fab fa-instagram mt-2"
        />
      </a>
      <a
        className="links-social"
        href="/"
      >
        <i
          aria-label="Social"
          className="fab fa-snapchat-square  mt-2"
        />
      </a>
      <a
        className="links-social"
        href="/"
      >
        <i
          aria-label="Social"
          className="fab fa-twitter  mt-2"
        />
      </a>
    </div>
    <ul
      className="list-inline"
    >
      <li
        className="list-inline-item"
      >
        <a
          href="/"
        >
          Home
        </a>
      </li>
      <li
        className="list-inline-item"
      >
        <a
          href="/"
        >
          Services
        </a>
      </li>
      <li
        className="list-inline-item"
      >
        <a
          href="/"
        >
          About
        </a>
      </li>
      <li
        className="list-inline-item"
      >
        <a
          href="/"
        >
          Terms
        </a>
      </li>
      <li
        className="list-inline-item"
      >
        <a
          href="/"
        >
          Privacy Policy
        </a>
      </li>
    </ul>
    <p
      className="copyright last-line"
    >
      RecipeDev Made With Love by
       
      <a
        className="text-primary"
        href="http://github.com/ceejayski"
      >
        Ceejayski 
      </a>
       
      Â© 2021
    </p>
  </footer>
</div>
`;
